schema/crucible-ci.json

```diff
@@ -0,0 +1,169 @@
+                                        "enum": [
+                                            "k8s",
+                                            "remotehosts"
+                                        ]
+                                    }
+                                },
+                                "runners": {
+                                    "oneOf": [
+                                        {
+                                            "type": "object",
+                                            "properties": {
+                                                "type": {
+                                                    "type": "string",
+                                                    "enum": [
+                                                        "github"
+                                                    ]
+                                                }
+                                            },
+                                            "additionalProperties": false,
+                                            "required": [
+                                                "type"
+                                            ]
+                                        },
+                                        {
+                                            "type": "object",
+                                            "properties": {
+                                                "type": {
+                                                    "type": "string",
+                                                    "enum": [
+                                                        "self-hosted"
+                                                    ]
+                                                },
+                                                "tags": {
+                                                    "type": "array",
+                                                    "minItems": 1,
+                                                    "uniqueItems": true,
+                                                    "items": {
+                                                        "type": "string",
+                                                        "minLength": 1
+                                                    }
+                                                }
+                                            },
+                                            "additionalProperties": false,
+                                            "required": [
+                                                "tags",
+                                                "type"
+                                            ]
+                                        }
+                                    ]
+                                }
+                            },
+                            "additionalProperties": false,
+                            "required": [
+                                "enabled",
```

Comment:
ok for being more explicit, this could be optional...

Reply:
Yeah, originally I was going to make this optional and then I changed my mind and decided to make it required so that, if necessary, one could just go into the file and very quickly and easily flip the value from True to False or vice versa without having to worry about the schema and where the `enabled` property was allowed and not.  I'm not absolutely tied to that decision, that is just how I got to where it is currently at.